### ISSUE TITLE
Incorrect matrix math in tutorial at:  http://www.tensorflow.org/tutorials/mnist/beginners/index.md

### ISSUE THREAD

**Issue by lazarillo** [tensorflow/tensorflow#115]
On the [generally wonderful beginner's documentation](http://www.tensorflow.org/tutorials/mnist/beginners/index.md), there is the following incorrect matrix math ![matrix math](http://api.tensorflow.org/system/image/body/1707/softmax-regression-scalarequation.png)

-----

The x's are incorrect.  Each element should include x{sub}1, x{sub}2, and x{sub}3.  This will be obvious upon looking at the correct vector representation on that same page immediately underneath.  Overall the documentation seems awesome so far!  :smiley:

**Comment by colah** [comment]
D'oh. I think I was half asleep when I made that. Here's a fixed image:

![softmax-regression-scalarequation](https://cloud.githubusercontent.com/assets/61658/11082384/5f4fb1ca-87d9-11e5-93d6-4130585466b8.png)

The repo should be fixed shortly.

**Comment by vrv** [comment]
colah just fixed this in f6d5a2caedd3982a1e6dcaa99bfafe0539527144.  Will take some time to push to the website though. Thanks for the report!

**Comment by lazarillo** [comment]
Thanks for the super fast response!  (And all the great work!)
